src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt

```diff
@@ -164,61 +164,61 @@ object PartyFinderStats {
 
                                 remove(null)
                             }
-                            component.append(
-                                UTextComponent("§dImportant Items: §7(Hover)\n\n").setHoverText(
-                                    items.joinToString(
-                                        "§8, "
-                                    ).ifBlank { "§c§lNone" }
-                                )
-                            )
+                            component.append(UTextComponent("§dImportant Items: §7(Hover)\n").setHoverText(items.joinToString(
+                                "§8, "
+                            ).ifBlank { "§c§lNone" }))
                         }
+                    val bloodMobsKilled = (profileData.player_stats.kills["watcher_summon_undead"]?.toInt()
+                        ?: 0) + (profileData.player_stats.kills["master_watcher_summon_undead"]?.toInt() ?: 0)
+
+                    component.append(
+                        UTextComponent("§5Miscellanous: §7(Hover)\n\n").setHoverText(
+                            """
+                                #§aTotal Secrets Found: §l§e${NumberUtil.nf.format(secrets)}
+                                #§aBlood Mobs Killed: §l§e${NumberUtil.nf.format(bloodMobsKilled)}
+                                #§dMagical Power: §l§e${profileData.accessory_data.highest_magical_power.toInt()}
```

Comment:
Not a big fan of that, too many people just snagged accessories for SB XP and then handed them back.

Reply:
ayo jani270, so ig I need to calculate it? ;-;

Reply:
> ayo jani270, so ig I need to calculate it? ;-;

If you want the correct MP, yes

Reply:
![Screenshot_20240220-115801.png](https://github.com/Skytils/SkytilsMod/assets/74718793/96bbfec7-95ab-43bc-8cb3-bea043477fa0)

Is this accurate currently btw?

Reply:
> ![Screenshot_20240220-115801.png](https://private-user-images.githubusercontent.com/74718793/306235728-96bbfec7-95ab-43bc-8cb3-bea043477fa0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0NjA0MDEsIm5iZiI6MTcwODQ2MDEwMSwicGF0aCI6Ii83NDcxODc5My8zMDYyMzU3MjgtOTZiYmZlYzctOTVhYi00M2JjLThjYjMtYmVhMDQzNDc3ZmEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIwVDIwMTUwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzMjI0OGE2Mjg1MDhkNDAwMjQzMjJlOWE4ODhhZGFmZmNkMTcwMzU3YmJiY2RkNTdkMDU0MjYxMWJhZmVlZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JakZOFUMam6xZB0PPpevuMU__9HOjUFPY4WtUYzYtU0)
> 
> Is this accurate currently btw?

indeed

Reply:
![image](https://github.com/Skytils/SkytilsMod/assets/74718793/251b289c-9138-4b5b-882e-233f02fa46c4)
current code (will be pushed once my hypixel-api code gets accepted)

Reply:
Wait wtf what a coincidence, I never saw your message

Reply:
Also note to self, add a command to trigger the message

Reply:
Also last thing for now, do I care about this? "Dungeon Accessories have double the Magical Power while in the Catacombs."

Reply:
> Also last thing for now, do I care about this? "Dungeon Accessories have double the Magical Power while in the Catacombs."

I would not